react/index.js

```diff
@@ -127,33 +120,13 @@ class ProductSummary extends Component {
       name: showFieldsProps,
     } = this.props
 
-    const classes = classNames(`${productSummary.container} overflow-hidden br3 h-100`, {
-      'flex flex-column justify-between center tc': displayMode !== 'inline',
-      [`${productSummary.containerNormal}`]: displayMode === 'normal',
-      [`${productSummary.containerSmall}`]: displayMode === 'small',
-      [`${productSummary.containerInline} w-100`]: displayMode === 'inline',
-    })
-
-    const linkClasses = classNames(`${productSummary.clearLink} flex`, {
-      'flex-column': displayMode !== 'inline',
-    })
-
-    const imageContainerClasses = classNames(`${productSummary.imageContainer} db`, {
-      'w-100 center': displayMode !== 'inline',
-      'w-30': displayMode === 'inline',
-    })
-
-    const informationClasses = classNames(`${productSummary.information}`, {
-      'w-70 pb2 pl3 pr3 flex flex-column justify-between': displayMode === 'inline',
-    })
-
-    const elementClasses = classNames(`${productSummary.element} pointer ph2 pt3 pb4 flex flex-column`, {
-      'bb b--muted-4 mh2 mt2': showBorders,
-    })
+    const displayModeMap = {
```

Comment:
this const can go to outside of the render component, around line 17.

Reply:
Fixed